library.js

```diff
@@ -1,5 +1,18 @@
 import specialCharacterList from './specialCharacters';
 
+function getRandomAlpha() {
+  const alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+  return alphabet.charAt(Math.floor(Math.random() * alphabet.length));
+}
+
+function getRandomDigit() {
+  return String(Math.round(Math.random() * 9));
```

Comment:
floor подходит как нельзя лучше. а вот 9 нужно исправить на 10 ;)

```suggestion
  return String(Math.floor(Math.random() * 10));
```